phpstan-baseline.neon

```diff
@@ -1065,6 +1065,106 @@ parameters:
 			count: 1
 			path: src/HandlerStack.php
 
+		-
+			message: "#^Property GuzzleHttp\\\\Idn\\:\\:\\$encodeTable has no typehint specified\\.$#"
```

Comment:
Can we exclude the file from PHPStan instead?

Reply:
Done.